### ISSUE TITLE
Does `webc:root` work with layouts?

### ISSUE THREAD

**Issue by urob** [11ty/eleventy-plugin-webc#112]
I am trying to use `webc:root` to add an attribute to the host component of a layout. For instance, in the following example I would like to add `class="some-class"` to `body` in the host layout:

`_include/base.webc`:
```html
<!doctype html>
<html>
  <head> ... </head>
  <body @raw="content"></body>
</html>
```

`page.webc`:
```html
---
layout: base.webc
---
<template class="some-class" webc:root>Some content</template>
```

Conceptually, this seems very similar to the documented purpose of `webc:root`. Maybe there is a way to make this work already now? Otherwise, this might be a useful feature to support.

**Comment by zachleat** [comment]
No, unfortunately [Eleventy layouts](https://www.11ty.dev/docs/layouts/) are an Eleventy specific feature (separate from WebC entirely) that is template syntax independent. _If_ you’d like access to this, don’t use Eleventy Layouts. Just use another WebC component directly!
